packages/react-ui-ag/src/Buttons/ApplyButton.js

```diff
@@ -0,0 +1,49 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { Button } from '@rentpath/react-ui-core'
+
+@themed(/^ApplyButton/, {
```

Comment:
If you're not using any other classes like `ApplyButton-foo` then you can just do:
```
@themed(['ApplyButton'], {
```

Reply:
I think you don't need the array IIRC. :+1: